modules/optimization/src/optimizationreporters/OptimizationReporterBase.C

```diff
@@ -110,27 +96,11 @@ OptimizationReporterBase::updateParameters(const libMesh::PetscVector<Number> &
 Real
 OptimizationReporterBase::getLowerBound(dof_id_type i) const
 {
-  return _lower_bounds.empty() ? std::numeric_limits<Real>::lowest()
-                               : _lower_bounds[getParameterIndex(i)];
+  return _lower_bounds[i];
```

Comment:
I started filling these completely so they don't need to be indexed into.  There are also always filled so there is no longer a need to check if bounds exist.